cluster/userdata-worker.yaml

```diff
@@ -60,11 +60,11 @@ coreos:
         --pod-manifest-path=/etc/kubernetes/manifests \
         --cluster_dns=10.3.0.10 \
         --cluster_domain=cluster.local \
-        --cloud-provider=aws \
         --kubeconfig=/etc/kubernetes/worker-kubeconfig.yaml \
         --require-kubeconfig \
         --tls-cert-file=/etc/kubernetes/ssl/worker.pem \
         --tls-private-key-file=/etc/kubernetes/ssl/worker-key.pem \
+        --cloud-provider=aws \
```

Comment:
Yes, this is the userdata for AWS clusters, I just moved the line so it would be less different from the vagrant version. :)